src/cli/info.rs

```diff
@@ -68,22 +87,94 @@ impl Display for Info {
-                writeln!(f, "  - {}", c)?;
+            if let Some(count) = pi.package_count {
+                writeln!(f, "{:20}: {}", "Dependency count", count)?;
+            }
+
+            if let Some(size) = &pi.cache_size {
+                writeln!(f, "{:20}: {}", "Cache size", size)?;
+            }
+
+            if let Some(update_time) = &pi.last_updated {
+                writeln!(f, "{:20}: {}", "Last updated", update_time)?;
+            }
+
+            if pi.platforms.len() > 0 {
+                for (i, p) in pi.platforms.iter().enumerate() {
+                    if i == 0 {
+                        writeln!(f, "{:20}: {}", "Target platforms", p)?;
+                    } else {
+                        writeln!(f, "{:20}: {}", "", p)?;
+                    }
+                }
+            }
+
+            if pi.tasks.len() > 0 {
+                for (i, t) in pi.tasks.iter().enumerate() {
+                    if i == 0 {
+                        writeln!(f, "{:20}: {}", "Tasks", t)?;
+                    } else {
+                        writeln!(f, "{:20}: {}", "", t)?;
+                    }
+                }
             }
         }
 
         Ok(())
     }
 }
 
+// Returns the size of a directory
+fn dir_size(path: impl Into<PathBuf>) -> miette::Result<String> {
+    fn dir_size(mut dir: fs::ReadDir) -> miette::Result<u64> {
+        dir.try_fold(0, |acc, file| {
+            let file = file.into_diagnostic()?;
+            let size = match file.metadata().into_diagnostic()? {
+                data if data.is_dir() => dir_size(fs::read_dir(file.path()).into_diagnostic()?)?,
+                data => data.len(),
+            };
+            Ok(acc + size)
+        })
+    }
+
+    let size = dir_size(fs::read_dir(path.into()).into_diagnostic()?)?;
+    Ok(format!("{} MiB", size / 1024 / 1024))
+}
+
+// Returns last update time of file, formatted: DD-MM-YYYY H:M:S
+fn last_updated(path: impl Into<PathBuf>) -> miette::Result<String> {
+    let time = fs::metadata(path.into())
+        .into_diagnostic()?
+        .modified()
+        .into_diagnostic()?;
+    let formated_time = DateTime::<Local>::from(time)
+        .format("%d-%m-%Y %H:%M:%S")
+        .to_string();
+
+    Ok(formated_time)
+}
+
+// Returns number of dependencies on current platform
+fn dependency_count(project: &Project) -> miette::Result<u64> {
+    let dep_count = project
+        .all_dependencies(Platform::current())?
+        .keys()
+        .cloned()
+        .fold(0, |acc, _| acc + 1);
+
+    Ok(dep_count)
+}
+
 pub async fn execute(args: Args) -> miette::Result<()> {
     let project = Project::load_or_else_discover(args.manifest_path.as_deref()).ok();
 
     let project_info = project.map(|p| ProjectInfo {
         manifest_path: p.root().to_path_buf().join("pixi.toml"),
         tasks: p.manifest.tasks.keys().cloned().collect(),
+        package_count: dependency_count(&p).ok(),
+        cache_size: dir_size(p.root().join(".pixi")).ok(),
```

Comment:
This is the size of the environment, not the size of the cache. Maybe the name of the variable should be `environment_size`?

Reply:
Makes sense, I'll change this.